huma.go

```diff
@@ -136,11 +137,14 @@ func findParams(registry Registry, op *Operation, t reflect.Type) *findResult[*p
 			pfi.Required = true
 		} else if q := f.Tag.Get("query"); q != "" {
 			pfi.Loc = "query"
-			name = q
+			split := strings.Split(q, ",")
+			name = split[0]
 			// If `in` is `query` then `explode` defaults to true. Parsing is *much*
-			// easier if we use comma-separated values, so we disable explode.
-			nope := false
-			explode = &nope
+			// easier if we use comma-separated values, so we disable explode by default.
+			if slicesContains(split[1:], "explode") {
```

Comment:
Consider using the built in slices package: https://pkg.go.dev/slices#Contains

Reply:
This is to support Go 1.20 FYI. Once Go 1.23 is released I will probably make the minimum required version 1.21 and we can start using the `slices` package.